keylime/cli/policies.py

```diff
@@ -25,12 +28,14 @@ def enforce_pcrs(tpm_policy, protected_pcrs, pcr_use):
             sys.exit(1)
 
 
-def process_allowlist(args):
+def process_policy(args):
```

Comment:
@mpeters At present, this function does also deal with `tpm_policy` and `mb_refstate` as well, so maybe it's most descriptive to leave it as is? A later refactor could pull those bits out into a separate function, and then it could be renamed.

Reply:
Nevermind then, I didn't catch that it also processed those other policy things.